### ISSUE TITLE
press space starts new experiment

### ISSUE THREAD

**Issue by thartbm** [thartbm/PyVMEC#56]
When pressing SPACE for the Xth time to end a pause trial, the last GUI button that was clicked gets activated. That is the button that starts a new experiment so you get a pop-up asking for a new participant ID, and there seems to be no way to get back to the actual experiment.

Marco ran himself on an experiment twice and each time at the same point he got this issue. That means we just can't run the experiments we want to run....

Possible solutions are changing the button (to C for 'continue'... or G for 'GO'?), or messing with the focussed window by telling PsychoPy to put focus on itself. Alternatively, all buttons in the PyVMEC GUI should be 'inactive' as long as an experiment runs...

**Comment by thartbm** [comment]
There is a function `psychopy.visual.Window._setCurrent()` that might do the trick. I see mentions of and `activate()` method of psychopy windows, but I can't find it.

**Comment by juliusjgm12** [comment]
I see,

I've added a module called pyautogui to the experiment.py that moves the mouse to a different location at the start of an experiment. Hopefully this will prevent any clicking issues from occurring. It is strange because on my set-up, the PyVMEC GUI is locked while a PsychoPy visual window is open.

This might just be a quick fix for now but we can test it out so that it'll at least allow for experiments to be run without errors?

I also disabled the run button while experiments run so it can't possibly disrupt a running experiment.
